packages/flutter/lib/src/services/text_input.dart

```diff
@@ -1126,6 +1126,10 @@ abstract class TextInputClient {
 
   /// Requests that the client remove the text placeholder.
   void removeTextPlaceholder() {}
+
+  /// Performs macOS specific selector from the NSStandardKeyBindingResponding
+  /// protocol.
+  void performSelector(String selectorName) {}
```

Comment:
One more thing I just thought of: I want to make sure that this is something that we want to add to TextInputClient and not split out.  We've been having a bunch of discussion about this topic, see this design doc: https://docs.google.com/document/d/1OxDsf_eot7TlsRn57z-1_dGbvuqnMxO7QEGIAmOCZtA/edit